File: Docker_2/js/fibonacci.js
function fibonacci(n) {
    let a = 0, b = 1;
    for (let i = 0; i < n; i++) {
        [a, b] = [b, a + b];
    }
    return a;
}

const start = Date.now();
const result = fibonacci(10000);
const end = Date.now();

console.log(end - start);

File: Docker_2/go/fibonacci.go
package main

import (
    "fmt"
    "time"
)

func fibonacci(n int) int {
    a, b := 0, 1
    for i := 0; i < n; i++ {
        a, b = b, a+b
    }
    return a
}

func main() {
    start := time.Now()
    result := fibonacci(10000)
    elapsed := time.Since(start)

    fmt.Println(elapsed.Milliseconds())
}
